.travis.yml

```diff
@@ -4,8 +4,11 @@ language: java
 
 jdk:
   - oraclejdk8
-  - oraclejdk7
-  - openjdk7
+
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
```

Comment:
Since the current master already contains this change it'd be nicer (and less conflicting) to rebase your changes there, instead of imitating changes in your own branch.